### PR TITLE
src/tutorials: Do minor text improvements

### PR DESCRIPTION
**Pull request by 413umc**

### CODE REVIEW

File: src/tutorials/hole_punching.rs [hunk 1]
```diff
@@ -148,3 +148,4 @@
 //!    ``` ignore
 //!    [2022-01-30T12:54:11Z INFO  client] DirectConnectionUpgradeSucceeded { remote_peer_id: PeerId("12D3KooWPjceQrSwdWXPyLLeABRXmuqt69Rg3sBYbU1Nft9HyQ6X") }
 //!    ```
+
```
Comment (mxinden) [inline src/tutorials/hole_punching.rs]: Why the newline?

**Review by mxinden — COMMENTED** [review]
Just one small comment. Looks good to me otherwise. Help is very much appreciated @413umc!

File: src/tutorials/hole_punching.rs (hunk 1 above)
Comment (413umc) [reply src/tutorials/hole_punching.rs]: This was accidental. I have removed it.

File: src/tutorials/ping.rs [hunk 2]
```diff
@@ -361,3 +365,4 @@
 //! [`PeerId`]: crate::core::PeerId
 //! [`Swarm`]: crate::swarm::Swarm
 //! [`development_transport`]: crate::development_transport
+
```
Comment (mxinden) [inline src/tutorials/ping.rs]:
```suggestion
```

**Review by mxinden — APPROVED** [review]
Thanks!

**Pull request merged**